### PR TITLE
Rimworld: UNSC Armoury

### PR DESCRIPTION
**Pull request by onigamii**
## Additions
Compatibility patch for [Rimworld: UNSC Armoury](https://steamcommunity.com/sharedfiles/filedetails/?id=2287902429)

## Testing
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [x] Playtested

### CODE REVIEW

File: Patches/HLX Rimworld UNSC Armoury/Weapons_Ranged.xml [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+          </statBases>
+          <costList>
+            <Steel>40</Steel>
+            <Plasteel>50</Plasteel>
+            <ComponentSpacer>5</ComponentSpacer>
+          </costList>
+          <Properties>
+            <verbClass>CombatExtended.Verb_ShootCE</verbClass>
+            <hasStandardCommand>true</hasStandardCommand>
+            <defaultProjectile>Bullet_12mmRailgun_Sabot</defaultProjectile>
```
Comment (N7Huntsman) [inline Patches/HLX Rimworld UNSC Armoury/Weapons_Ranged.xml:378]: I'd look at either the 6mm or 8mm. The 12mm is basically more for vehicle-mounted weapons, almost.

File: Patches/HLX Rimworld UNSC Armoury/Weapons_Ranged.xml [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+            <SwayFactor>1.23</SwayFactor>
+            <Bulk>6</Bulk>
+            <Mass>5</Mass>
+            <RangedWeapon_Cooldown>0.36</RangedWeapon_Cooldown>
+          </statBases>
+          <Properties>
+            <recoilAmount>1.35</recoilAmount>
+            <verbClass>CombatExtended.Verb_ShootCE</verbClass>
+            <hasStandardCommand>true</hasStandardCommand>
+            <defaultProjectile>Bullet_762x54mmR_FMJ</defaultProjectile>
```
Comment (N7Huntsman) [inline Patches/HLX Rimworld UNSC Armoury/Weapons_Ranged.xml:186]: Why not make this the same caliber as the Assault Rifle?

File: Patches/HLX Rimworld UNSC Armoury/Apparel_Headgear.xml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    </mods>
+    <match Class="PatchOperationSequence">
+      <operations>
+
+        <!-- === Heavy Combat Helmet === -->
+        <li Class="PatchOperationReplace">
+          <xpath>/Defs/ThingDef[defName="RUA_Apparel_CombatHelmet"]/statBases/StuffEffectMultiplierArmor</xpath>
+          <value>
+            <Bulk>6</Bulk>
+            <WornBulk>4</WornBulk>
```
Comment (N7Huntsman) [inline Patches/HLX Rimworld UNSC Armoury/Apparel_Headgear.xml:16]: This seems too high. Even the power armor helmets only have 1 wornBulk.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
A couple of suggestions for changes.

File: Patches/HLX Rimworld UNSC Armoury/Weapons_Ranged.xml (hunk 1 above)
Comment (onigamii) [reply Patches/HLX Rimworld UNSC Armoury/Weapons_Ranged.xml:378]: I went first with 6mm but it felt that it wasn't doing much damage so i changed to 12mm but maybe you're right and 6mm is the best.

File: Patches/HLX Rimworld UNSC Armoury/Weapons_Ranged.xml (hunk 2 above)
Comment (onigamii) [reply Patches/HLX Rimworld UNSC Armoury/Weapons_Ranged.xml:186]: Technically in-lore the Battle Rifle uses **9.4x40mm** as opposed to the AR / DMR that use **7.62x51mm** and honestly i didn't want to make a new caliber so i just slapped this in.

File: Patches/HLX Rimworld UNSC Armoury/Apparel_Headgear.xml (hunk 3 above)
Comment (onigamii) [reply Patches/HLX Rimworld UNSC Armoury/Apparel_Headgear.xml:16]: Whoops my bad

File: Patches/HLX Rimworld UNSC Armoury/Weapons_Ranged.xml (hunk 2 above)
Comment (N7Huntsman) [reply Patches/HLX Rimworld UNSC Armoury/Weapons_Ranged.xml:186]: Ah, okay. I may got ahead and add a new caliber. :P

**Review by N7Huntsman — APPROVED** [review]
Good to go. I'll add the Battle Rifle ammo separately.

**Pull request merged**